app/src/main/java/com/hedvig/app/feature/offer/ui/OfferActivity.kt

```diff
@@ -154,22 +160,35 @@ class OfferActivity : BaseActivity(R.layout.activity_offer) {
                 .viewState
                 .flowWithLifecycle(lifecycle)
                 .onEach { viewState ->
+                    binding.progressBar.isVisible = viewState is OfferViewModel.ViewState.Loading
+                    binding.offerScroll.isVisible = viewState !is OfferViewModel.ViewState.Loading
+                    when (viewState) {
+                        is OfferViewModel.ViewState.Loading -> {}
```

Comment:
show progress?

Reply:
The functionality is the same as it was before the changes. The lines 163-164 are handling the loading state so there's nothing to put inside this case.